osu.Game.Rulesets.Mania/Timing/TimingSection.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Game.Beatmaps.Timing;
+
+namespace osu.Game.Rulesets.Mania.Timing
+{
+    /// <summary>
+    /// A point in the map where the beat length or speed multiplier has changed .
+    /// </summary>
+    public class TimingSection

```

Comment:
This should probably derive from `ControlPoint` at very least, since you only require one field not contained in there.